### PR TITLE
Update explaining syncing and bi-directional syncing

### PR DESCRIPTION
**Pull request by deniseschannon**
Updated https://deploy-preview-329--vcluster-docs-site.netlify.app/docs/vcluster/next/configure/vcluster-yaml/sync/

Also updated but the content is duplicated from above.
https://deploy-preview-329--vcluster-docs-site.netlify.app/docs/vcluster/next/configure/vcluster-yaml/sync/to-host
https://deploy-preview-329--vcluster-docs-site.netlify.app/docs/vcluster/next/configure/vcluster-yaml/sync/from-host

### CODE REVIEW

**Review by sowmyav27 — CHANGES_REQUESTED** [review]

File: vcluster/_fragments/how-does-syncing-work.mdx [hunk 1]
```diff
@@ -1,9 +1,37 @@
...
+extra permissions to sync, which are automatically given to the vCluster ServiceAccount if you enable a resource that requires additional 
+permissions. 
 
-vCluster only syncs low-level resources by default, such as pods, secrets, configmaps or services. As soon as one of these resources are created in the virtual cluster, vCluster will sync it to the other side. As soon as a resource was copied by vCluster, it will start watching for changes to the resource on either side, so when a controller changes something on the host cluster side, the vCluster will sync that change to the virtual cluster or when a change on the virtual cluster side happens, it will sync it to the host cluster, to make sure the resource is in sync.
+All fields of a resources are synced from one direction to the other based on what section the resource is part of in the `vcluster.yaml`. There 
+are a few exceptions for each resource where specific fields are kept in sync in both directions (i.e. bi-directional sync). 
+As soon as a resource is copied by vCluster, the syncer starts watching for changes to the resource on either side. When the resource
```
[the remark this answers is not in the document]
Comment (deniseschannon) [reply vcluster/_fragments/how-does-syncing-work.mdx:11]: The original sentence made it imply that bi-directional syncing was happening, which is not the case, but I'll try to clear it up a bit.

File: vcluster/_fragments/how-does-syncing-work.mdx [hunk 2]
```diff
@@ -1,9 +1,37 @@
...
+is updated in the origin cluster, the syncer updates and copies those changes in the destination cluster. 
+
+When a resource is defined in the `toHost` section, as soon as that resource is created in the virtual cluster, vCluster copies it to the host cluster. 
+Any changes to the resource in the virtual cluster are automatically applied to the resource in the host cluster.
```
[the remark this answers is not in the document]
Comment (deniseschannon) [reply vcluster/_fragments/how-does-syncing-work.mdx:15]: No, changes in the host cluster are not automatically applied back to the resource in the virtual cluster. It's only the specific fields, which I've put in the parts of syncing when we talk about resources being synced to/from Host

File: vcluster/_fragments/how-does-syncing-work.mdx [hunk 3]
```diff
@@ -1,9 +1,40 @@
-Syncing is the process where vCluster copies a resource between the virtual cluster and the host cluster. It's the basic principle vCluster uses to emulate a fully functional Kubernetes cluster.
+Since a virtual cluster does not have actual worker nodes or a network, syncing is the process where vCluster copies a resource between the virtual cluster and the host cluster. 
+It's the basic principle that vCluster uses to emulate a fully functional Kubernetes cluster. 
 
+By default, vCluster only syncs low-level resources, such as pods, secrets, configmaps, or services, but a user can 
+enable many other resources for all use cases. vCluster control plane runs with a minimal set of RBAC permissions to allow execution in restricted environments. Certain resources require 
+extra permissions to sync, which are automatically given to the vCluster ServiceAccount if you enable a resource that requires additional 
+permissions.
```
[the remark this answers is not in the document]
Comment (deniseschannon) [reply vcluster/_fragments/how-does-syncing-work.mdx:7]: I didn't realize this was also used in the architecture page. 

I'll review the duplicate content in the architecture page vs the "how does syncing work" to see if we want to reuse the fragment in both places or link.

File: vcluster/_fragments/how-does-syncing-work.mdx [hunk 4]
```diff
@@ -1,9 +1,40 @@
...
+All fields of a resources are synced from either the virtual cluster to the host cluster or the inverse (i.e. from the host cluster to the virtual cluster). The direction on how
+the sync of the resources is based on what section the resource is part of in the `vcluster.yaml`. There 
+are a few exceptions for each resource where specific fields are kept in sync in both directions (i.e. bi-directional sync). 
+
+If the resource is being asked to sync from the virtual cluster to the host cluster, vCluster copies the resources from the vCluster and sends it to the host cluster to be created. There are 
+a couple of additional labels added to the host cluster resource object for vCluster to keep it in sync. The syncer starts watching for changes to the resource on either side. When the resource
+is updated in the virtual cluster, the syncer copies those changes in host cluster.
```
[the remark this answers is not in the document]
Comment (deniseschannon) [reply vcluster/_fragments/how-does-syncing-work.mdx:15]: Good point, I'll write/clarify this. 

Basically, changes in the host cluster are overwritten whenever the virtual cluster resource is synced to the host cluster unless it's a bi-directional field. 

I'll need to check how often we sync.

File: vcluster/_fragments/sync-from-host-resources.mdx [hunk 5]
```diff
@@ -1,17 +1,34 @@
...
+A good example would be nodes, which are nice to view inside the virtual cluster and can be also used to enabled certain features such as [scheduling inside the vCluster](/vcluster/configure/vcluster-yaml/control-plane/other/advanced/virtual-scheduler.mdx), 
+but you wouldn't want your virtual cluster to change the node itself. Another benefit of only syncing from host is that vCluster itself only requires read-only RBAC permissions. 
 
 vCluster also allows to sync custom resources via the [custom resource definitions syncer](../configure/vcluster-yaml/sync/from-host/custom-resources.mdx)
 
-### Enabled by Default
+There are a couple of labels that are created on the host cluster by vCluster that never get synced to the virtual cluster resource. These labels are:
```
[the remark this answers is not in the document]
Comment (deniseschannon) [reply vcluster/_fragments/sync-from-host-resources.mdx]: I need to clarify on what happens as it wasn't clear with the original docs. 

I know from virtual -> host, we create a set of new labels on the host cluster, that don't exist on the virtual cluster. 

On host -> virtual, I am not sure what labels we add in and when. I'll need to clarify and check.

**Pull request merged**